scripts/bootstrap.bash

```
@@ -0,0 +1,13 @@
+# Get workspace dependencies
+flutter pub get
+
+# For each package in packages that has build_runner dependency run build_runner
+for dir in packages/*; do
+  if [ -f "$dir/pubspec.yaml" ]; then
+    if grep -q build_runner "$dir/pubspec.yaml"; then
+      pushd $dir
```

Shellcheck (0.10.0)

[warning] Use 'pushd ... || exit' or 'pushd ... || return' in case pushd fails.

(SC2164)